labels/mapping.go

```diff
@@ -35,8 +61,10 @@ var helpWanted = []Label{
 	{"Good First Issue", "Suitable for first-time contributors", "7057ff"},
 	{"Hacktoberfest", "", "dc7d02"},
 	{"Help Wanted", "Community help wanted", "33aa3f"},
-	{"Needs spec", "Needs further specification to be a good help wanted ticket", "88acea"},
+	// {"Needs spec", "Needs further specification to be a good help wanted ticket", "88acea"},
 	{"Tech/Go", "", "0e8a16"},
+	{"Tech/JavaScript", "", "f9d0c4"},
+	{"Tech/TypeScript", "", "c9ffff"},
```

Comment:
I'm would like to keep things simple and just have `Tech/ReactJS`. What is the value of distinguishing between the two types?

Reply:
TypeScript is in fashion...

Reply:
/shrug-wick

Reply:
/shrug_wic

Reply:
Is this really a reason?

Reply:
Yes. If you find it ridiculous, I will not disagree, and will remove the label.

Reply:
Please remote it. Let's go with just the two "Tech"s for now.